docs/troubleshooting_pi-server_integration.rst

```diff
@@ -0,0 +1,180 @@
+.. Images
+.. |img_001| image:: images/tshooting_pi_001.jpg
+.. |img_002| image:: images/tshooting_pi_002.jpg
+.. |img_003| image:: images/tshooting_pi_003.jpg
+.. |img_004| image:: images/tshooting_pi_004.jpg
+.. |img_005| image:: images/tshooting_pi_005.jpg
+.. |img_006| image:: images/tshooting_pi_006.jpg
+.. |img_007| image:: images/tshooting_pi_007.jpg
+.. |img_008| image:: images/tshooting_pi_008.jpg
+.. |img_009| image:: images/tshooting_pi_009.jpg
+.. |img_010| image:: images/tshooting_pi_010.jpg
+
+Troubleshooting the PI-Server integration
+=========================================
+
+This section describes how to trouble shoot issues with the PI-Server integration
+using Fledge version >= 1.9.x and PI Web API 2019 SP1 1.13.0.6518
+
+- Log files
+- How to check the PI Web API is installed and running
+- Commands to check the PI Web API
+- Error messages and the related causes
+- Possible solutions to common situations
+
+Log files
+---------
+
+Fledge logs into the system syslog, mainly warnings and errors and in some circumstances rows of severity information.
```

Comment:
Fledge logs messages at error and warning levels by default, it is possible to increase the verbosity of messages logged to include information and debug messages also. This is done by altering the minimum log level setting for the north service or task. To change the minimal log level within the graphical user interface select the north service or task, click on the advanced settings link and then select a new minimal log level from the option list presented.

Reply:
done